WordPress/WordPressTest/BlogTests.swift

```diff
@@ -305,4 +305,48 @@ final class BlogTests: CoreDataTestCase {
             try? XCTAssertNotNil(Blog.lookup(withID: 123, in: context))
         }
     }
+
+    func testWhenAccountNilDoesSupportRestApiFalse() {
+        let blog = BlogBuilder(mainContext).build()
+
+        XCTAssertNil(blog.account)
+        XCTAssertFalse(blog.supportsRestApi())
+    }
+
+    func testWhenAccountNotNilDoesSupportRestApiTrue() {
+        let blog = BlogBuilder(mainContext).withAnAccount().build()
+
+        XCTAssertNotNil(blog.account)
+        XCTAssertTrue(blog.supportsRestApi())
+    }
+
+    func testSupportsStockPhotos() {
```

Comment:
I'd suggest avoiding writing tests for configuration. In this case, the tests are more complicated than the implementation and they match it one-to-one.